contracts/starknet/lib/voting.cairo

```diff
@@ -880,9 +880,10 @@ func unchecked_add_voting_strategies{
         # The following elements are the actual params
         unchecked_add_voting_strategy_params(next_index, 1, params_len, params)
 
+        Voting_num_voting_strategies_store.write(next_index + 1)
+
```

Comment:
So now that I think about it, even though starknet should, in theory, only count a single `write` operation even if we do multiple ones (because they are modifying the same address), currently that's not the case.
I think we should pass `next_index` as a parameter to this function, and wrap this function inside another one that does something like:
```
read the index store
call wrapped function to add strategies
write the new index (should be `prev_index + addresses_len`)
```

Reply:
The same thought crossed my mind but I cba at the time lol. will do though